src/platform/qt/AudioDevice.h

```diff
@@ -20,7 +20,9 @@ Q_OBJECT
 
 	void setInput(mCoreThread* input);
 	void setFormat(const QAudioFormat& format);
-	bool atEnd() const override;
+	virtual bool isSequential() const override;
+	virtual bool atEnd() const override;
+	virtual qint64 bytesAvailable() const override;
```

Comment:
Nit: `virtual` is redundant when `override` is applied, so I've not been putting it in new code.

```suggestion
	bool isSequential() const override;
	bool atEnd() const override;
	qint64 bytesAvailable() const override;
```